src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Jobs/GetAzureRmRecoveryServicesBackupJob.cs

```diff
@@ -117,6 +117,10 @@ public override void ExecuteCmdlet()
                 {
                     throw new Exception(Resources.JobAllowedDateTimeRangeExceeded);
                 }
+                else if(rangeStart > DateTime.UtcNow)
```

Comment:
We do one time manual testing before checkin the changes for the checks, we don't have automated cases for that.